api/src/main/java/io/minio/PostPolicy.java

```diff
@@ -91,6 +92,17 @@ public void setContentType(String contentType) throws InvalidArgumentException {
     this.contentType = contentType;
   }
 
+  /**
+   * Sets success action status.
+   */
+  public void setSuccessActionStatus(int successActionStatus) throws InvalidArgumentException {
+    if (!(successActionStatus == 200 || successActionStatus == 201 || successActionStatus == 204)) {
+      throw new InvalidArgumentException("Success action status accepts the values as 200, 201, or "
```

Comment:
```suggestion
      throw new InvalidArgumentException("Invalid action status, acceptable values are 200, 201, or 204"
```

Reply:
Now to thinking more, we should have successActionStatus as enum value such as

```
successActionStatus enum {
   successActionStatus201
   successActionStatus200
   successActionStatus204
}
```

This way validation becomes easy.

Reply:
done.